src/components/Header.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[];
+  completeAll: () => void;
+  activeCount: number;
+  handleSubmit: (event: React.FormEvent) => void;
+  inputRef: React.RefObject<HTMLInputElement>;
+  newTodoTitle: string;
+  setNewTodoTitle: React.Dispatch<React.SetStateAction<string>>;
```

Comment:
will be easier, what do u think?
```suggestion
  setNewTodoTitle: (title: string) => void;
```